torch/_torch_docs.py

```diff
@@ -145,21 +145,22 @@ def merge_dicts(*dicts):
            r"""
 .. function:: add(input, other, out=None)
 
-Adds the scalar :attr:`other` to each element of the input :attr:`input`
-and returns a new resulting tensor.
+Each element of the tensor :attr:`other` is added to each element of the
+tensor :attr:`input`.
 
 .. math::
     \text{{out}} = \text{{input}} + \text{{other}}
 
-If :attr:`input` is of type FloatTensor or DoubleTensor, :attr:`other` must be
-a real number, otherwise it should be an integer.
+The shapes of :attr:`input` and :attr:`other` must be
+:ref:`broadcastable <broadcasting-semantics>`.
 
-Args:
-    {input}
-    value (Number): the number to be added to each element of :attr:`input`
+If the :class:`torch.dtype` of :attr:`input` and :attr:`other` differ,
```

Comment:
does this "if" add anything?  The the rules of type promotion apply even if the dtypes are the same, right?